### ISSUE TITLE
LRS3 433h pretrain configuration

### ISSUE THREAD

**Issue by jxzhanggg** [facebookresearch/av_hubert#41]
Hi,

I'm following your work on LRS3 433h, starting from MFCC feature.
However, I found  finetuned (LRS3 30h) lipreading WER is high, even when I finished the 4th iter (WER 81%).

I suppose the reason is the first 4 pretrain iter, modality drop is 0, causing mismatch during pretraining (always using 
audio + video) and finetuning (using video).
Could you please check if the configuration is right? 

Thx!

**Comment by chevalierNoir** [comment]
Hi,

The modality dropout in the first 4 iterations is 0 and we only use set it to non-zero in the last iteration. We are going to run the same experiment and check the pre-training config. Just for sanity check, did you tune the hyperparameters in finetuning? The main hyperparameters we tune are `optimization.max_update` and `model.freeze_finetune_updates`. We mostly train for `{12K, 15K, 30K}` steps and freeze it for either 100% or 80% training steps. The learning rate is linearly warmup for 1/3 of whole training steps.

**Comment by jxzhanggg** [comment]
I used finetune/base_lrs3_30h.yaml and didn't tune the hyperparameters. I finetuned your released ckpt of 4th iter with the
same configuration. Resulting WER is 52%, which looks normal.

**Comment by jxzhanggg** [comment]
FYI,
My masked valid accuracy  ("valid_correct_m_0") is 
iter1(mfcc feature, 100 class), 42.5%,
iter2 (iter1 hubert 9 layer, 100 class),  62.7%
iter3 (iter2 hubert 12 layer, 500 class), 60.2%
iter3 (iter3 hubert 12 layer, 1000 class), 60.0%
